tests/test_plugin_features.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import pytest
+
+import dclab
+from dclab.rtdc_dataset.plugins import PlugInFeature
+
+from helper_methods import retrieve_data
+
+
+@pytest.fixture
```

Comment:
Currently a placeholder for a fixture similar to your temporary feature fixture.

Reply:
Yes, just call `remove_all_plugin_features()` at the bottom and set `autouse=True` so every test automatically gets it.